office-365-management-api/office-365-management-activity-api-schema.md

```diff
@@ -2233,3 +2237,47 @@ The audit records for events related to Purview Governance use this schema (in a
 |OldValue |JObject |No |The old value of the object before change.|
 |NewValue |JObject |No |The New/current value of the object after change.|
 
+## Viva Access Management
+
+Viva Access Management comprises of the schema related to  policy management for various viva features. It includes action performed by the tenant IT Admin to create, modify, and delete the policy for various viva features. It uses t the [Common schema](#common-schema) with the following record types.
+
+### Enum Type AllowedModifiedFields
+
+|**Value**|**Member name**|**Description**|
+|:-----|:-----|:-----|
+|1|PolicyName|Attribute Policy|
+|2|ACL|Access Control List. This attribute suggests that either the identity was added or removed from the policy.|
+|3|IsEnabled|Attribute showing state of policy allow/deny.|
+
+### VfamCreatePolicy record type
+
+|**Properties**|**Type**|**Mandatory?**|**Description**|
+|:-----|:-----|:-----|:-----|
+|ModuleId|Edm.String|Yes|Viva module Id for which the policy is being created.|
+|FeatureId|Edm.String|Yes|Viva feature Id for which the policy is being created. feature is part of module.|
+|PolicyId|Edm.String|Yes| Policy Id of newly created policy for above module and feature ids.|
+|IsEnabeled|Edm.Boolean|Yes| Status showing if the policy is enabled or disabled for the given feature.|
+|AddedIdentities|Collection(Edm.String)|Yes| List of user/group OIDs that are part of the created policy
+
+### VfamUpdatePolicy record type
+
+|**Properties**|**Type**|**Mandatory?**|**Description**|
+|:-----|:-----|:-----|:-----|
+|ModuleId|Edm.String|Yes|Viva module Id for which the policy is being created.|
+|FeatureId|Edm.String|Yes|Viva feature Id for which the policy is being created. feature is part of module.|
+|PolicyId|Edm.String|Yes| Policy Id of newly created policy for above module and feature ids.|
+|IsEnabeled|Edm.Boolean|Yes| Status showing if the policy is enabled or disabled for the given feature.|

```

Comment:
Typo in `IsEnabled` Also the policies themselves aren't enabled or disabled right? May want to clarify the desciption